### PR TITLE
feat(content) Change `require` to `on offer` conditions in Lunarium Intro

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork / Missions / Jobs)**

## Summary
Changing the `on offer` `require` conditions to `to offer` `has` conditions for outfits in the Lunarium intro, similar to https://github.com/endless-sky/endless-sky/pull/9358.

### CODE REVIEW

File: data/coalition/lunarium intro.txt [hunk 1]
```diff
@@ -792,6 +792,7 @@ mission "House Bebliss 5-FW"
 	landing
 	to offer
 		has "House Bebliss 4-FW: done"
+		has "outfit: Jump Drive" 2
```
Comment (ziproot) [inline data/coalition/lunarium intro.txt:795]:
```suggestion
		"outfit: Jump Drive" >= 2 
```

Comment (Saugia) [reply data/coalition/lunarium intro.txt]: Thanks, I forgot that it being a condition here isn't the same thing.

File: data/coalition/lunarium intro.txt [hunk 2]
```diff
@@ -820,9 +820,10 @@ mission "House Bebliss 5-FW"
 		"assisting lunarium" ++
 	on visit
 		dialog `You land on <planet>, but you haven't brought the extra jump drive you needed to yet. Make sure to have it in your cargo before you land.`
+	to complete
+		"outfit: Jump Drive" >= 2
```
Comment (ziproot) [inline data/coalition/lunarium intro.txt:824]:
```suggestion
		"outfit: Jump Drive" >= 2
```
My bad.

Comment (Saugia) [reply data/coalition/lunarium intro.txt:824]: Np.

**Review by ziproot — APPROVED** [review]

**Review by lumbar527 — APPROVED** [review]

**Pull request merged**